javascript/intermediate/multiple-cash-registers_sylvain1811.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Hacktoberithms - Multiple Cash Registers
+ * https://github.com/the-vampiire/hacktoberithms/issues/68
+ * By sylvain1811, 7 oct 2019
+ * node version v12.9.1
+ */
+
+function checkout_time(customers, n_cashier) {
+    let cashiers = Array(n_cashier).fill(0)
+    customers.forEach(customer => {
+        minIndex = cashiers.reduce((accIdx, current, index) => {
```

Comment:
consider declaring this as a `const` or `let` instead of as a global variable (for scope safety)

Reply:
You're right, I forgot about that! Thanks for your feedbak!